src/main/java/tasks/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package tasks;
+public class Deadline extends Task{
+    protected String time;
+    protected String desc;
```

Comment:
consider refactoring desc to task class
also, spell out full variable if it helps readibility